app/assets/javascripts/application.js

```diff
@@ -19,4 +19,5 @@
 //= require rails-ujs
 //= require turbolinks
 //= require moment/moment
-//= require_tree .
+//= require clipboard/dist/clipboard
+//= require_tree .
```

Comment:
Looks like you removed the line at the end of the file, can you set your text editor to not do this?

Reply:
(and remove that particular change)

Reply:
Aight, done.